packages/editor/web/statics/styles/side-toolbar.scss

```
@@ -45,13 +45,14 @@
   width: 320px;
   max-height: 400px;
   display: grid;
-  box-shadow: 0 4px 8px 0 rgba(123, 99, 99, 0.1), 0 0 4px 0 rgba(0, 0, 0, 0.1);
+  box-shadow: $settings-whitebox-box-shadow;
+  border-radius: $settings-whitebox-border-radius;
   pointer-events: auto;
   overflow: hidden;
 }
 
 .plusIcon {
-  color: $settings-action-color;
+  color: $settings-icons-color;
```

Revert change back to 

```suggestion
  color: $settings-icons-color;
```